tests/init_db.py

```diff
@@ -23,8 +38,11 @@ def init(self):
         self._import_forest_perimeters()
 
     def _truncate(self):
-        from pyramid_oereb.standard.models import main, contaminated_sites, \
-             land_use_plans, motorways_building_lines, forest_perimeters
+        from pyramid_oereb.standard.models import main
```

Comment:
This function looks somehow hard to maintain to me is it not possible to do something similar to what you did in `create_tables.py` but instead of drop truncate the tables?

Reply:
I understand what you mean-But I think this will go away asap when the testing and sample data is joined. I would like to keep this out of scope for this PR-do you agree?

Reply:
Yes that is fine for me.